.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-Fixes: <https://github.com/mozilla/addons/ISSUENUM>
+Fixes: <mozilla/addons/issues/ISSUENUM>
```

Comment:
To properly render it should either be

`<https://github.com/mozilla/addons/issues/ISSUENUM>`

Or

`mozilla/addons#ISSUENUM` (no `<`/`>`)

Reply:
See https://gist.github.com/diox/0528bf196dee01cba51d3b0c572226a7